### ISSUE TITLE
Breaking block texture glitch

### ISSUE THREAD

**Issue by teodor256** [minetest/minetest#14275]
![Screenshot from 2024-01-18 20-56-12](https://github.com/minetest/minetest/assets/156880388/d62464b2-586c-4ac3-88f2-17526a69965d)
### Minetest version

```true
Minetest 5.6.1 (Linux)
Using Irrlicht 1.9.0mt8
Using LuaJIT 2.1.0-beta3
BUILD_TYPE=RelWithDebInfo
RUN_IN_PLACE=0
USE_CURL=1
USE_GETTEXT=1
USE_SOUND=1
STATIC_SHAREDIR="/usr/share/games/minetest"
STATIC_LOCALEDIR="/usr/share/locale"
```

### Active renderer

OpenGL 4.6

### Operating system and version

Debian GNU/Linux 12

### OpenGL version

4.6

### Summary

The breaking texture remains on block after player stop breaking it by opening inventory.

### Steps to reproduce

Choose creative mode.
Start breaking blocks continuously.
Press "E" to open inventory while breaking.
After pressing "E" again to exit, block would have breaking texture. It can remains still or disappear after couple of seconds.

**Comment by teodor256** [comment]
![Uploading Screenshot from 2024-01-18 20-56-12.png…]()

**Comment by Zughy** [comment]
Can't reproduce on 5.8. Please update your version to 5.8 and tell us if you can still reproduce it

**Comment by grorp** [comment]
Duplicate of #5611, fixed by #13336.
